least_common_paths.txt

```diff
@@ -0,0 +1,992 @@
+                      './tests/rule_based_profiler/data_assistant/test_data_assistant_result.py',
+                      './tests/rule_based_profiler/data_assistant/test_onboarding_data_assistant.py',
+                      './tests/rule_based_profiler/data_assistant/test_volume_data_assistant.py'],
+            'requirements_file': 'requirements.txt'},
+ 'boto3': {'common_path': '.',
+           'files': ['./great_expectations/checkpoint/util.py',
+                     './great_expectations/cli/upgrade_helpers/upgrade_helper_v11.py',
+                     './great_expectations/cli/v012/upgrade_helpers/upgrade_helper_v11.py',
+                     './great_expectations/data_context/store/tuple_store_backend.py',
+                     './great_expectations/data_context/util.py',
+                     './great_expectations/datasource/batch_kwargs_generator/s3_batch_kwargs_generator.py',
+                     './great_expectations/datasource/data_connector/configured_asset_s3_data_connector.py',
+                     './great_expectations/datasource/data_connector/inferred_asset_s3_data_connector.py',
+                     './great_expectations/datasource/pandas_datasource.py',
+                     './great_expectations/execution_engine/pandas_execution_engine.py',
+                     './tests/actions/conftest.py',
+                     './tests/data_context/store/test_html_site_store.py',
+                     './tests/data_context/store/test_store_backends.py',
+                     './tests/data_context/store/test_validations_store.py',
+                     './tests/data_context/test_data_context_in_code_config.py',
+                     './tests/datasource/batch_kwarg_generator/test_s3_generator.py',
+                     './tests/datasource/data_connector/test_configured_asset_dbfs_data_connector.py',
+                     './tests/datasource/data_connector/test_configured_asset_s3_data_connector.py',
+                     './tests/datasource/data_connector/test_inferred_asset_dbfs_data_connector.py',
+                     './tests/datasource/data_connector/test_inferred_asset_s3_data_connector.py',
+                     './tests/datasource/test_pandas_datasource.py',
+                     './tests/execution_engine/conftest.py',
+                     './tests/integration/docusaurus/deployment_patterns/aws_emr_serverless_deployment_patterns.py',
+                     './tests/integration/docusaurus/deployment_patterns/aws_glue_deployment_patterns.py',
+                     './tests/integration/usage_statistics/test_integration_usage_statistics.py'],
+           'requirements_file': 'requirements-dev-lite.txt'},
+ 'colorama': {'common_path': 'great_expectations/cli',
+              'files': ['./great_expectations/cli/cli.py',
+                        './great_expectations/cli/v012/cli.py'],
+              'requirements_file': 'requirements.txt'},
```

Comment:
`colorama` only impacts the CLI but we define it in our global `requirements.txt`.

Can we perhaps make CLI installation optional or an add-on? 

Other libaries like FastAPI have a particular syntax that is quite handy:
```
pip install "fastapi[all]"
```

If `cli` was a submodule or some kind of optional dep, we could trim down our installation size.